### DISCUSSION TITLE
How does a TreeView bind events via MVVM

### DISCUSSION THREAD

**Discussion by Hub-moon** [AvaloniaUI/Avalonia#13240] [Q&A]
How does the AvaloniaUi TreeView bind events with MVVM and get node information from the events

**Comment by pyking-wang** [comment]
Just pay attention to the property changes of the SelectedItem in the treeview.

**Comment by workgroupengineering** [comment]
Using [Avalonia.Xaml.Behaviors](https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors) like [this ](https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors/blob/46e516a0a11a349528bc19051fbcaafaf7b29609/tests/Avalonia.Xaml.Interactions.UnitTests/Core/InvokeCommandAction002.axaml#L10)
